subprojects/diagnostics/src/main/java/org/gradle/api/plugins/HelpTasksPlugin.java

```diff
@@ -72,6 +82,14 @@ public void apply(final Project project) {
             task.setDescription("Displays the outgoing variants of " + projectName + ".");
             task.setGroup(HELP_GROUP);
             task.setImpliesSubProjects(true);
+            task.getShowAll().convention(false);
+        });
+        tasks.register(RESOLVABLE_CONFIGURATIONS_TASK, ResolvableConfigurationsReportTask.class, task -> {
+            task.setDescription("Displays the configurations available to resolve in " + projectName + ".");
```

Comment:
```suggestion
            task.setDescription("Displays the configurations that can be resolved in " + projectName + ".");
```
I think this wording flows better.